index.js

```diff
@@ -1,18 +1,10 @@
 // @flow
 
 import React from 'react';
-import {
-  PanResponder,
-  View,
-  Dimensions,
-  Animated,
-  TouchableWithoutFeedback,
-} from 'react-native';
+import {Animated, Dimensions, PanResponder, TouchableWithoutFeedback, View} from 'react-native';
 import PropTypes from 'prop-types';
 import styles from './styles';
 
-type WindowDimensions = { width: number, height: number };
```

Comment:
Any reasons to remove it?

Reply:
Not sure why I did this, but I added it back in.